app/controllers/concerns/foreman/controller/parameters/discovered_host.rb

```diff
@@ -0,0 +1,20 @@
+module Foreman::Controller::Parameters::DiscoveredHost
+  extend ActiveSupport::Concern
+  include Foreman::Controller::Parameters::HostBase
+  include Foreman::Controller::Parameters::HostCommon
+
+  class_methods do
+    def discovered_host_params_filter
+      Foreman::ParameterFilter.new(::Host::Discovered, :top_level_hash => 'host').tap do |filter|
```

Comment:
> :top_level_hash isn't accepted here any more, this second arg should be able to be removed.  (And it should now work for your API controller.)

Thanks for headsup, I will fix that and test your PR tomorrow.

-- 
Later,
 Lukas #lzap Zapletal